docs/source/getting-started/tabular-data/identifier-truncation.rst

```diff
@@ -0,0 +1,294 @@
+Identifier Truncation
+=====================
+
+OpenDP can be used to query datasets where each individual may
+contribute an unbounded number of records, but where all records
+contributed by an individual share the same identifier.
+
+The user identifier is a part of the privacy unit (the privacy unit
+quantifies the influence an individual may have on the data).
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: python
+
+            >>> import opendp.prelude as dp
+            >>> import polars as pl
+            
+            >>> # the PIDENT column contains individual identifiers
+            >>> # an individual may contribute data under at most 1 PIDENT identifier
+            >>> privacy_unit = dp.unit_of(contributions=1, identifier=pl.col("PIDENT"))
+            
+
+This ``privacy_unit`` consists of all records associated with any one
+unique identifier in the ``PIDENT`` column. OpenDP allows identifiers to
+be arbitrary Polars expressions. The identifier expression must be
+row-by-row to be well-defined.
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: python
+
+            >>> dp.enable_features("contrib")
+            
+            >>> context = dp.Context.compositor(
+            ...     data=pl.scan_csv(dp.examples.get_france_lfs_path(), ignore_errors=True),
+            ...     privacy_unit=privacy_unit,
+            ...     privacy_loss=dp.loss_of(epsilon=1.0, delta=1e-8),
+            ...     split_evenly_over=4,
+            ...     margins=[dp.polars.Margin(max_length=150_000 * 36)],
+            ... )
+            
+
+Truncating Per-Group Contributions
+----------------------------------
+
+In order to make differentially private releases on this data, an
+additional identifier truncation step is necessary, where only a limited
+number of records corresponding to each identifier are kept.
+
+Under the assumption that it unlikely that an individual is chosen for
```

Comment:
```suggestion
Under the assumption that it is unlikely that an individual is chosen for
```

Reply:
Thanks: I'll pick this up in the next iteration, just so this doesn't need another CI run.